### PR TITLE
Comments

### PR DESCRIPTION
**Pull request by lidiawisniewska**
Our users want to comment on movies. Let's add feature that allows them to do that.

We should have new comment form under the movie description and all the comments listed below that.

Additionally - one user should be allowed to leave only one comment to the movie. He can create another only if he deletes previous one.

### CODE REVIEW

File: app/decorators/comment_decorator.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class CommentDecorator < Draper::Decorator
+  delegate_all
+
+  # Define presentation-specific methods here. Helpers are accessed through
```
Comment (mpiask) [inline app/decorators/comment_decorator.rb]: Try not to leave unnecessary comments.

**Comment by lidiawisniewska** [comment]
I’m away today and will be back tomorrow late afternoon. Would it be okay if I resolve these by end of tomorrow?

**Comment by mpiask** [comment]
For now it's OK, just remember those suggestions.

**Comment by lidiawisniewska** [comment]
Cool, thank you.

**Pull request merged**